lib/api/listParts.js

```diff
@@ -136,11 +139,50 @@ function listParts(authInfo, request, log, callback) {
                     return next(null, destBucket, mpuBucket, mpuOverview);
                 });
         },
-        function waterfall3(destBucket, mpuBucket, mpuOverviewArray, next) {
+        function waterfall3(destBucket, mpuBucket, mpuOverview, next) {
             // BACKWARD: Remove to remove the old splitter
             if (mpuBucket.getMdBucketModelVersion() < 2) {
                 splitter = constants.oldSplitter;
             }
+
+            if (config.backends.data === 'multiple') {
+                // Reconstruct mpuOverviewKey to serve
+                // as key to pull metadata originally stored when mpu initiated
+                const mpuOverviewKey =
+                    `overview${splitter}${objectKey}${splitter}${uploadId}`;
+
+                return metadata.getObjectMD(mpuBucket.getName(), mpuOverviewKey,
```

Comment:
if you refactor services.metadataValidateMultipart to be a get of the mpuOverviewKey instead of a listing you can avoid this additional call.  It was a listing for reasons that no longer apply (the key used to contain more information).  That function would be much clearer if it was just a get and then you would have everything you need without an extra call to metadata.